### PR TITLE
Fix typo in samqfs doc #224

### LINKED ISSUE #224: Typos in the "Storage Archive Manager handbook" (samqfs.md)
While reading, I encountered these typos:
>It is reommended to save the metadata of the filesystem (inodes) for situations where you have to recover the SamFS filesystem.

* [samqfs.md, line 320](https://github.com/OpenIndiana/oi-docs/blob/master/docs/handbook/community/samqfs.md?plain=1#L320): 's/reommended/recommended/'

>To much CLI for you?

* [samqfs.md, line 326](https://github.com/OpenIndiana/oi-docs/blob/master/docs/handbook/community/samqfs.md?plain=1#L326): 's/To/Too/'

I'm not sure if this is an improvement, but:
'There is a more comfortable termbased tool /opt/SUNWsamfs/sbin/samu.'
can be written as:
'There is a more comfortable TUI-based tool /opt/SUNWsamfs/sbin/samu.'

### PR DESCRIPTION
**Pull request by JMadgwick**
Fixes #224

### CODE REVIEW

**Comment by JMadgwick** [comment]
@domag02

File: docs/handbook/community/samqfs.md [hunk 1]
```diff
@@ -317,13 +317,13 @@ If an attempt is made to open a file which is offline the stager will copy the d
...
 
-It is reommended to save the metadata of the filesystem (inodes) for situations where you have to recover the SamFS filesystem.
+It is recommended to save the metadata of the filesystem (inodes) for situations where you have to recover the SamFS filesystem.
 
 With assistance of such metadata dumps you are able to recover older versions of the files also.
 
 ## SamFS administration
 
-To much CLI for you? There is a more comfortable termbased tool `/opt/SUNWsamfs/sbin/samu`.
+To much CLI for you? There is a more comfortable terminal tool available: `/opt/SUNWsamfs/sbin/samu`.
```
Comment (domag02) [inline docs/handbook/community/samqfs.md:326]: Is this not a problem?:
's/To much/Too much/'?

Comment (JMadgwick) [reply docs/handbook/community/samqfs.md]: Yes, I missed it. Fixed now.

**Review by domag02 — APPROVED** [review]
Thanks for the fix.
LGTM.

**Pull request merged**